### ISSUE TITLE
[Misc]: Missing cu118 wheels for 0.6.2 release

### ISSUE THREAD

**Issue by tderrmann** [vllm-project/vllm#9021]
Hello,

For version 0.6.2, the cu118 wheels are not available on the release page:
[0.6.2 release page](https://github.com/vllm-project/vllm/releases/tag/v0.6.2)

While 0.6.1.post2 has them:
[0.6.1.post2 release page](https://github.com/vllm-project/vllm/releases/tag/v0.6.1.post2)

Are you planning on continuing to release those wheels or should we start building vllm+cu118 from source?

**Comment by tderrmann** [comment]
I'm wondering if it's related to the failures in the [Create Release action](https://github.com/vllm-project/vllm/actions/workflows/publish.yml)?

**Comment by tarekabouzeid** [comment]
Yes, I think its this build specifically : https://github.com/vllm-project/vllm/actions/runs/11041663331/job/30672417507
